.gitignore

```diff
@@ -4,3 +4,6 @@ data/etfs/
 data/stocks/
 data/symbols_valid_meta.csv
 obsidian-vault-apache-spark/
+**/__pycache__/
+*.png
```

Comment:
put these files into a dedicated local directory; e.g. 'local-files' and exclude this directory; 
In case you want to add pngs to your readme, you want to be able to add them.